README.md

```diff
@@ -686,6 +686,8 @@ Table of Contents
   * [zoneedit.com](https://www.zoneedit.com/free-dns/) — Free DNS hosting with Dynamic DNS Support.
   * [zonewatcher.com](https://zonewatcher.com) — Automatic backups and DNS change monitoring. 1 domain free
   * [huaweicloud.com](https://www.huaweicloud.com/intl/en-us/product/dns.html) – Free DNS hosting by Huawei
+  * [alibabacloud.com](https://www.alibabacloud.com/product/dns) — Free DNS hosting by Alibaba Cloud
```

Comment:
I cant see details of what the free edition actually gives you so its a bit hard to include this, do you have a link I might have missed somewhere?

Reply:
If you're asking about DNS itself, more information can be found [here](https://www.alibabacloud.com/help/doc-detail/102232.htm).

Reply:
yeah, I dont see anything that tells you what the free features are - just tht if you stop paying some features stopped working and you only get the free features.  But what are those?

Reply:
From what the support team told me: 

1. Minimum TTL: 600 seconds
2. Unlimited requests
3. Four nodes in China
3. Unlimited subdomains.
4. Two forwarding.
5. Load balancing
6. Support IPv6.
7. Level 5 subdomains are supported
8. Unlimited number of domains
9. Intelligent analysis

They also noticed that all this information is on the Chinese version of the site, but not on the international one and attached a link to the [Chinese version of the site](https://help.aliyun.com/document_detail/100426.html).

Reply:
OK thanks, afraid I cant take this one then I need to be able to verify things easily. So lets remove this one and the other is good to go in.

Reply:
No problems. Done.